lib/Backend/ServerScriptContext.cpp

```diff
@@ -63,11 +63,15 @@ ServerScriptContext::~ServerScriptContext()
 #endif
     if (m_asmJsInterpreterThunkBufferManager)
     {
+        m_asmJsInterpreterThunkBufferManager->Decommit();
```

Comment:
do you think there is benefit to decommit first? we are just going to free right after this so seems like maybe not necessary